api/src/middlewares/services/UserService/personalisation/personalisationBuilder/userPostalConfirmation/index.ts

```diff
@@ -18,17 +17,9 @@ export function buildUserPostalConfirmationPersonalisation(answers: AnswersHashM
   const isSuccessfulPayment = metadata.payment?.state?.status === "success" ?? false;
   const country = answers["country"] as string;
   const post = answers["post"] as string;
-  const previousMarriage = answers.maritalStatus && answers.maritalStatus !== "Never married";
-  const getAdditionalCountryPersonalisation = postalPersonalisationsByCountry[country];
-
-  const additionalPersonalisations = {
-    ukProofOfAddressNeeded: false,
-    spainProofOfAddressNeeded: false,
-    croatiaCertNeeded: false,
-    italySpainPartnerPreviousMarriageDocNeeded: false,
-    showSpainContent: false,
-    ...getAdditionalCountryPersonalisation?.(answers),
-  };
+  const previousMarriage = answers.maritalStatus !== "Never married";
+  const livesAbroad = answers.livesInCountry === false;
```

Comment:
Can we be more verbose with this? I think it's getting confusing now because we use the term differently now. Abroad now means doesn't live in the UK, or not in the application country